### ISSUE TITLE
Player/Pick projected bid amount

### ISSUE THREAD

**Issue by elitracy** [cwendt94/espn-api#503]
### Sport

Basketball

### Summary

As a user, I want to be able to compare the projected bid amount to the actual bid amount as well as other stats so that I can better determine player value.

**Comment by cwendt94** [comment]
Is this request for the draft or getting players from wavier wire?

**Comment by elitracy** [comment]
I'm hoping to see how much they were valued at by ESPN during the draft. So their projected draft amount.

**Comment by cwendt94** [comment]
Im not sure if that information is available after drafting. I just checked the draft API response from ESPN and it only has bid amount. I don't know where else that information would be. Are you able to find it on their website at all?

**Comment by elitracy** [comment]
Yes during the fantasy season when I scroll down on the player page I can see how much they were drafted by and by which team.

**Comment by cwendt94** [comment]
I see that as well but I don't see their projected bid amount.
